eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -263,6 +264,11 @@ datumDatahash = \case
   (DatumHash d) -> SJust d
   (Datum d) -> SJust $ hashData d
 
+datumFromDatahash :: StrictMaybe (DataHash (Crypto era)) -> Datum era
```

Comment:
this function name is a bit confusing, it looks like it converts a data hash to a data. and hilariously the type looks like it converts a data hash to a data. :) of course, it's really just converting an alonzo data hash to a babbage data hash. I'm terrible at names myself. maybe `translateDataHash`? I'm not sure if we would need this function outside of the translation module, maybe this function should live there.